Ms2Database/Program.cs

```diff
@@ -4,25 +4,22 @@
 using System.Text;
 using System.Threading.Tasks;
 using Ms2Database.DbClasses;
+using Ms2Database.Controllers;
 
 namespace Ms2Database
 {
     public class Program
     {
         static void Main(string[] args)
         {
-            using (Ms2DbContext context = new Ms2DbContext())
-            {
-                Account account = new Account()
-                {
-                    Username = "localhost",
-                    Password = ""
-                };
-                context.Accounts.Add(account);
-                context.SaveChanges();
-            }
+            AccountManager accmanage = new AccountManager();
```

Comment:
Variable style according to coding style guidelines, use `accManage` or `accountManager`. I personally don't recommend abbreviating, it can make some things harder to understand, although it is pretty clear in this case.